src/ceph-detect-init/CMakeLists.txt

```diff
@@ -10,3 +10,10 @@ add_dependencies(tests ceph-detect-init)
 
 include(Distutils)
 distutils_install_module(ceph_detect_init)
+
+add_custom_target(ceph-detect-init-flake8
+  COMMAND
+  flake8 ceph_detect_init tests
+  WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/src/ceph-detect-init
+  COMMENT "syntax check")
+add_dependencies(ceph_detect_init ceph-detect-init-flake8)
```

Comment:
@tchaikov I'd like to express that `ceph-detect-init-flake8` should run on make all. Any idea on how this should be done ?